### PR TITLE
871760 - Added UG content of Custom View and Z ordering in ImageEditor control

### PR DESCRIPTION
**Pull request by devanathan-vaithiyanathan**
### Description
* Added UG content for custom view annotation.
* Added UG content for Z-ordering.

### CODE REVIEW

File: MAUI/ImageEditor/customview.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        Image customImage = new Image() { HeightRequest = 100, WidthRequest = 100, Aspect= Aspect.Fill};
+        customImage.Source = ImageSource.FromFile("emoji.png");
+        imageEditor.AddCustomAnnotationView(customImage);
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Customize the custom view appearance
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/customview.md:38]: Customize the appearance

File: MAUI/ImageEditor/customview.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+Customize the appearance of Custom view annotation using the [`ImageEditorAnnotationSettings`].
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/customview.md:40]: add link for annotation settings

File: MAUI/ImageEditor/customview.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        this.imageEditor.DeleteAnnotation();
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Clear all custom view
+
+Remove all the custom view annotations using the [`ClearAnnotations`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ClearAnnotations) method.
+
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/customview.md:107]: add note

N> This is common for Shape, Text and CustomView annotations.

Comment (karthikraja-arumugam) [reply MAUI/ImageEditor/customview.md:107]: update this note in other documents also, add custom view in existing note

File: MAUI/ImageEditor/customview.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+    private void OnClearAnnotationsClicked(object sender, EventArgs e)
+    {
+        this.imageEditor.ClearAnnotations();
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Annotation selected event
+
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/customview.md:131]: We have included annotation unselected event, mention that in all annotations UG

File: MAUI/ImageEditor/z-ordering.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+   {
+      this.imageEditor.BringToFront();
+   }
+
+{% endhighlight %}
+{% endtabs %}
+
+## SendToBack
+
+The `SendToBack` method is used to send the selected annotation to the back of a group of elements over an image.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/z-ordering.md:47]: The `SendToBack` method is used to send the selected annotation to the back of all the annotations.

File: MAUI/ImageEditor/z-ordering.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+   {
+      this.imageEditor.SendToBack();
+   }
+
+{% endhighlight %}
+{% endtabs %}
+
+## BringForward
+
+The `BringForward` method is used to bring the selected annotation to one step front of a group of elements over an image.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/z-ordering.md:73]: The `BringForward` method is used to bring the selected annotation to one step forward over the image.

File: MAUI/ImageEditor/z-ordering.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+   {
+      this.imageEditor.BringForward();
+   }
+
+{% endhighlight %}
+{% endtabs %}
+
+## SendBackward
+
+The `SendBackward` method is used to send the selected annotation to one step backward of a group of elements over an image.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/z-ordering.md:99]: The `SendBackward` method is used to send the selected annotation one step backward over an image.

File: MAUI/ImageEditor/z-ordering.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+                Clicked="OnSendBackwardClicked" />
+    </Grid>  
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+   private void OnSendBackwardClicked(object sender, EventArgs e)
+   {
+      this.imageEditor.SendBackward();
+   }
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/z-ordering.md:118]: In ImageEditorToolbarItem we have included a property ShowTooltip, which can be used to show or hide a tooltip for an ImageEditorToolbarItem. Mention this at the end of Toolbar document

File: MAUI/ImageEditor/z-ordering.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+The image editor control allows to change the position of annotations that are arranged over the editor. This can be achieved using the following methods:
+
+1. BringToFront
+2. SendToBack
+3. BringForward
+4. SendBackward
+
+## BringToFront
+
+The `BringToFront` method is used to bring the selected annotation to the front of a group of elements over an image.
```
[the remark this answers is not in the document]
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/z-ordering.md:21]: updated

File: MAUI/ImageEditor/z-ordering.md (hunk 5 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/z-ordering.md:47]: updated

File: MAUI/ImageEditor/z-ordering.md (hunk 6 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/z-ordering.md:73]: updated

File: MAUI/ImageEditor/z-ordering.md (hunk 7 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/z-ordering.md:99]: updated

File: MAUI/ImageEditor/customview.md (hunk 1 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/customview.md:38]: title updated

File: MAUI/ImageEditor/customview.md (hunk 2 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/customview.md:40]: link added for annotation settings

File: MAUI/ImageEditor/customview.md (hunk 4 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/customview.md:131]: annotation unselected content updated in all annotations UG.

File: MAUI/ImageEditor/z-ordering.md (hunk 8 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/z-ordering.md:118]: added

File: MAUI/ImageEditor/customview.md [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+{% endtabs %}
+
+
+## Customize the appearance
+
+Customize the appearance of Custom view annotation using the [`ImageEditorAnnotationSettings`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html).
+
+* [`AllowDrag`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_AllowDrag): Enables or disables the dragging of custom view annotation. The default value is `true`.
+* [`AllowResize`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_AllowResize): Enables or disables the resizing of custom view annotation. The default value is `true`.
+* [`Bounds`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_Bounds): Specifies the bounds of the custom view. Position the custom view wherever you want on the image. The value of the text bounds should fall between 0 and 1.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/customview.md:44]: The value of the view bounds should fall between 0 and 1.

File: MAUI/ImageEditor/toolbar-customization.md [hunk 11]
```diff
@@ -620,4 +620,28 @@ The default toolbar item action can be restricted by setting the `e.cancel` to `
...
+To show or hide the tooltip, set the [`ShowTooltip`] property to either true or false. By default, the [`ShowTooltip`] property is set to `true`.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+
+ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/toolbar-customization.md:636]: add one example

ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
if (headerToolbar.ToolbarItems.FirstOrDefault() is ImageEditorToolbarGroupItem browseGroup)
{
    ImageEditorToolbarItem? browseItem = browseGroup.Items.FirstOrDefault();
    if (browseItem != null)
    {
        browseItem.ShowTooltip = false;
    }
}

File: MAUI/ImageEditor/customview.md (hunk 10 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/customview.md:44]: updated

File: MAUI/ImageEditor/toolbar-customization.md (hunk 11 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/ImageEditor/toolbar-customization.md:636]: updated

**Review by karthikraja-arumugam — APPROVED** [review]
This is fine

**Pull request merged**